vexim/config/header.php

```diff
@@ -1,26 +1,27 @@
 <?php
-  if (isset($_SESSION['domain_id'])) {
-    $domheaderquery = "SELECT enabled FROM domains WHERE domains.domain_id=:domain_id";
-    $domheaderresult = $dbh->prepare($domheaderquery);
-    $domsuccess = $domheaderresult->execute(array(':domain_id'=>$_SESSION['domain_id']));
-    if ($domsuccess) { $domheaderrow = $domheaderresult->fetch(); }
-    $usrheaderquery = "SELECT enabled FROM users WHERE localpart=:localpart AND domain_id=:domain_id";
-    $usrheaderresult = $dbh->prepare($usrheaderquery);
-    $usrsuccess = $usrheaderresult->execute(array(':localpart'=>$_SESSION['localpart'], ':domain_id'=>$_SESSION['domain_id']));
-    if ($usrsuccess) { $usrheaderrow = $usrheaderresult->fetch(); }
+  if (isset($_SESSION['domain_id'])) { // Check if accound or domain haven't been disabled
+    $headerquery = "SELECT domains.enabled AS domain, users.enabled AS user FROM users,domains
+                    WHERE users.localpart=:localpart AND domains.domain_id=:domain_id AND users.domain_id=domains.domain_id;";
+    $headerresult = $dbh->prepare($headerquery);
+    $headersuccess = $headerresult->execute(array(':localpart'=>$_SESSION['localpart'], ':domain_id'=>$_SESSION['domain_id']));
+    if ($headersuccess) {
+      $headerrow = $headerresult->fetch();
```

Comment:
Nah, I mean we should probably have an `if (!$headersuccess)` case to handle possible issues, and then also check if `$headerrow` is not `false` (or is an array) before using it as an array.